_guidelines-v4/04-cmn/03-cmnadditionalmeasurecontent/08-cmndir.md

```diff
@@ -1,24 +1,7 @@
 ---
 sectionid: cmnDir
-title: "Directions and Rehearsal marks"
+title: "Directives and Rehearsal marks"
 version: "v4"
 ---
 
 In CMN scores, there is often a large number of natural language instructions. Some of them concern the loudness and the speed of the performance, in which case MEI offers the elements {% include link elem="dynam" %} (described at {% include link id="cmnDynam" %}) and {% include link elem="tempo" %}. In other cases, however, they provide other instructions for the performer. Instead of providing separate elements for all possible types of such directions, MEI offers the generic {% include link elem="dir" %} element. Although this element is not CMN specific (it is defined in {% include link id="shared" %}), it is especially important in this repertoire.
```

Comment:
Rather than saying _it is especially important_, I would say _it is frequently used_ in this repertoire. 
```suggestion
In CMN scores, there is often a large number of natural language instructions. Some of them concern the loudness and the speed of the performance, in which case MEI offers the elements {% include link elem="dynam" %} (described at {% include link id="cmnDynam" %}) and {% include link elem="tempo" %}. In other cases, however, they provide other instructions for the performer. Instead of providing separate elements for all possible types of such directions, MEI offers the generic {% include link elem="dir" %} element. Although this element is not CMN specific (it is defined in {% include link id="shared" %}), it is frequently used in this repertoire.
```